docs/content/stable/explore/observability/active-session-history.md

```diff
@@ -16,7 +16,11 @@ type: docs
 
 Active Session History (ASH) provides a current and historical view of system activity by sampling session activity in the database. A database session or connection is considered active if it is consuming CPU, or has an active RPC call that is waiting on one of the wait events.
 
-ASH exposes session activity in the form of [SQL views](../../ysql-language-features/advanced-features/views/) so that you can run analytical queries, aggregations for analysis, and troubleshoot performance issues. To run ASH queries, you need to enable [YSQL](../../../api/ysql/).
+ASH exposes session activity in the form of [SQL views](../../ysql-language-features/advanced-features/views/) so that you can run analytical queries, aggregations for analysis, and troubleshoot performance issues.
+
+To run ASH queries, you need to [enable YSQL](../../../reference/configuration/yb-tserver/#enable-ysql).
+
+To enable YSQL for YugabyteDB Anywhere universes, refer to [Enable database authorization](../../../yugabyte-platform/security/authorization-platform/#enable-database-authorization).
```

Comment:
This link is YBA specific.... ysql is enabled by default either way. Would replace with something like

Note that to run ASH queries, regardles of whether you are using YSQL or YCQL, the YSQL API must be enabled on your universe (the default).

Reply:
of course. right will change it to keep it specific this way